GreateCommonDivisor/GreateCommonDivisor/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace GreateCommonDivisor
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter the number1");
+            var number1 = int.Parse(Console.ReadLine());
+
+            Console.WriteLine("Enter the number2");
+            var number2 = int.Parse(Console.ReadLine());
+
+            var result = IntegerExtensions.FindGcd(number1, number2);
```

Comment:
Extension word could be misleading here. In my opinion, it would be better either rename it to 'helper' or implement real extension method (with 'this' keyword). Let's discuss it on the next lesson